Tests/FunctionalTests/PluginTests.swift

```diff
@@ -645,10 +646,14 @@ class PluginTests: XCTestCase {
             XCTAssertNoDiagnostics(observability.diagnostics)
             XCTAssert(packageGraph.packages.count == 1, "\(packageGraph.packages)")
             XCTAssert(packageGraph.rootPackages.count == 1, "\(packageGraph.rootPackages)")
-            let package = try XCTUnwrap(packageGraph.rootPackages.first)
+            let package: ResolvedPackage = try XCTUnwrap(packageGraph.rootPackages.first)
             
             // Find the regular target in our test package.
-            let libraryTarget = try XCTUnwrap(package.targets.map(\.underlyingTarget).first{ $0.name == "MyLibrary" } as? SwiftTarget)
+            let libraryTarget = try XCTUnwrap(
+                package.targets
+                    .map(\.underlyingTarget)
+                    .first{ $0.name == "MyLibrary" } as? SwiftTarget
+            )
```

Comment:
In both these cases, `XCTUnwrap` wasn't properly inferring the the type passes to it so I just helped it out. The failures were occurring locally on Xcode 14.